mmv1/products/firebase/ProjectLocation.yaml

```diff
@@ -39,16 +39,10 @@ skip_delete: true
 skip_sweeper: true
 deprecation_message: >-
   `google_firebase_project_location` is deprecated in favor of explicitly configuring `google_app_engine_application`
-  and `google_firestore_database`. This resource will be removed in the next major release of the provider.
-examples:
-  - !ruby/object:Provider::Terraform::Examples
-    name: 'firebase_project_location_basic'
```

Comment:
Should we also delete `mmv1/templates/terraform/examples/firebase_project_location_basic.tf.erb`?

Reply:
Oops, yes thanks!